templates/webpack.angular.js

```diff
@@ -46,11 +47,18 @@ module.exports = env => {
     const appFullPath = resolve(projectRoot, appPath);
     const appResourcesFullPath = resolve(projectRoot, appResourcesPath);
 
-    const entryModule = aot ?
-        nsWebpack.getAotEntryModule(appFullPath) : 
-        `${nsWebpack.getEntryModule(appFullPath)}.ts`;
+    const entryModule = `${nsWebpack.getEntryModule(appFullPath)}.ts`;
```

Comment:
Please remove the `getAotEntryModule` function from index.js as well.

Reply:
It's a breaking change if we remove it completely.

Reply:
Yep. We discussed to introduce a `Deprecated` decorator in another PR.